spinalcordtoolbox/deepseg/models.py

```diff
@@ -256,7 +263,16 @@ def install_model(name_model):
     :return: None
     """
     logger.info("\nINSTALLING MODEL: {}".format(name_model))
-    download.install_data(MODELS[name_model]['url'], folder(name_model))
+    url_field = MODELS[name_model]['url']
+    # List of mirror URLs corresponding to a single model
+    if isinstance(url_field, list):
+        model_urls = url_field
+        download.install_data(model_urls, folder(name_model))
+    # Dict of lists, with each list corresponding to a different model seed for ensembling
+    elif isinstance(url_field, dict):
```

Comment:
> (Otherwise, a bad url field would just cause this function to silently return success.)

Good call! The assertion would only be seen during SCT development (e.g. if someone adds a faulty entry to the `MODELS` dict), so this seems very appropriate.